src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -1708,7 +1708,7 @@ public static void DCJS_ReadOnlyDictionary()
         dict["Foo"] = 1;
         dict["Bar"] = 2;
         ReadOnlyDictionary<string, int> value = new ReadOnlyDictionary<string, int>(dict);
-        var deserializedValue = SerializeAndDeserialize(value, @"{""_dictionary"":[{""Key"":""Foo"",""Value"":1},{""Key"":""Bar"",""Value"":2}]}");
+        var deserializedValue = SerializeAndDeserialize(value, @"{""m_dictionary"":[{""Key"":""Foo"",""Value"":1},{""Key"":""Bar"",""Value"":2}]}");
```

Comment:
Can you please remove the `[SkipOnTargetFramework]` on the test? The test is being skipped on netfx. Removing the attribute would enable the test on netfx so that we get desktop<-->core coverage..